package.json

```diff
@@ -100,6 +100,7 @@
     "image-webpack-loader": "^4.2.0",
     "jest": "^22.4.0",
     "mini-css-extract-plugin": "^0.4.0",
+    "new-relic-source-map-webpack-plugin": "1.1.0",
```

Comment:
FYI: I pinned this to a specific version so we can be a bit more attentive to changes.